forge-gui/res/cardsfolder/upcoming/swine_rebellion.txt

```diff
@@ -1,15 +1,12 @@
 Name:Swine Rebellion
 ManaCost:2 G
 Types:Sorcery
-A:SP$ Branch | BranchConditionSVar$ X | BranchConditionSVarCompare$ GE3 | TrueSubAbility$ DBConjureAll | FalseSubAbility$ DBChoose | SpellDescription$ If you control three or more Boars with different names, conjure each card from the Three Pigs spellbook onto the battlefield.\nIf you control two or fewer Boars with different names, conjure two cards of your choice from the Three Pigs spellbook into your hand, then put one of them onto the battlefield.
-SVar:DBConjureAll:DB$ MakeCard | Conjure$ True | Names$ First Little Pig,Second Little Pig,Third Little Pig | Zone$ Battlefield 
-SVar:DBChoose:DB$ GenericChoice | Choices$ ConjureFirstandSecond,ConjureSecondandThird,ConjureFirstandThird
-SVar:ConjureFirstandSecond:DB$ MakeCard | Conjure$ True | Names$ First Little Pig,Second Little Pig | RememberMade$ True | Zone$ Hand | SubAbility$ DBPutBattlefield | SpellDescription$ Conjure First Little Pig and Second Litte Pig into your hand
-SVar:ConjureSecondandThird:DB$ MakeCard | Conjure$ True | Names$ Second Little Pig,Third Little Pig | RememberMade$ True | Zone$ Hand | SubAbility$ DBPutBattlefield | SpellDescription$ Conjure Second Little Pig and Third Litte Pig into your hand
-SVar:ConjureFirstandThird:DB$ MakeCard | Conjure$ True | Names$ First Little Pig,Third Little Pig | RememberMade$ True | Zone$ Hand | SubAbility$ DBPutBattlefield | SpellDescription$ Conjure First Little Pig and Third Litte Pig into your hand
-SVar:DBPutBattlefield:DB$ ChangeZone | Origin$ Hand | Destination$ Battlefield | ChangeType$ Creature.IsRemembered | ChangeNum$ 1 | Mandatory$ True | SelectPrompt$ Select a card for the battlefield | SubAbility$ DBCleanup
+A:SP$ Branch | BranchConditionSVar$ X | BranchConditionSVarCompare$ GE3 | TrueSubAbility$ DBConjureAll | FalseSubAbility$ DBConjureTwo | SpellDescription$ If you control three or more Boars with different names, conjure each card from the Three Pigs spellbook onto the battlefield. If you control two or fewer Boars with different names, conjure two cards of your choice from the Three Pigs spellbook into your hand, then put one of them onto the battlefield.
+SVar:DBConjureAll:DB$ MakeCard | Conjure$ True | Names$ First Little Pig,Second Little Pig,Third Little Pig | Zone$ Battlefield
+SVar:DBConjureTwo:DB$ MakeCard | Conjure$ True | Spellbook$ First Little Pig,Second Little Pig,Third Little Pig | SpellbookAmount$ 2 | RememberMade$ True | Zone$ Hand | SubAbility$ DBPutBattlefield
```

Comment:
How is that on Arena? does it allow choosing the same pig twice?

Reply:
Just tested this on Arena. 

It does not allow choosing the same twice

Reply:
well in that case the new implementation does it wrong?

Reply:
easy enough